Abha_Rana/Tic-Tac-Toe/tt.css

```diff
@@ -0,0 +1,32 @@
+* {
+    box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+  }
+  .container {
+    display: flex;
+    background: #fff;
+    align-content: center;
+    justify-content: center;
+    flex-direction: column;
+    min-width: 700px;
```

Comment:
You are using px as unit,  :)
Is the container responsive?

Reply:
used percent instead of px for responsiveness.